common/utils.c

```diff
@@ -239,7 +239,6 @@ char *str_lowering(const void *ctx, const char *string TAKES)
 	char *ret;
 
 	ret = tal_strdup(ctx, string);
-	for(char *p = ret; *p; p++) *p=tolower(*p);
-	tal_free(string);
+	for (char *p = ret; *p; p++) *p=tolower(*p);
```

Comment:
Ah, good, please fold into previous commit!

Reply:
yeah it was in a fixup commit just testing the code in the CI